server/services/participants-helper.js

```diff
@@ -56,6 +56,34 @@ const addDistanceToParticipantFields = (raw, siteDistanceJoin) =>
     }),
   }));
 
+// Find the previous status for each org, create a copy of it 
+const  revertToOldStatus = async ( participantStatuses,setParticipantStatus)=>{
+  await participantStatuses.forEach(async (status)=>{
+    if(status?.data?.previousStatus && status.data.previousStatus !=='archived' ){
+      const res = await setParticipantStatus(status.employer_id,status.participant_id, status.data.previousStatus,{
+        ...status.data?.previousData|| {}
+      } )
+    }
+  })
+  
+}
+// Withdraw the participant 
+const insertWithdrawalParticipantStatus = async (participantStatuses,setParticipantStatus) =>{ 
+  participantStatuses.forEach( async (status)=>{
+    // Prevent locking the user into a loop of archived statuses
+    if(status.status && status.status ==='archived'){
+      return;
+    }else{
+      await setParticipantStatus(status.employer_id,status.participant_id, 'archived',{
+        final_status: 'Withdrawn by MoH',
```

Comment:
Should we make this more generic? Thinking we'll probably reuse this method when participants can withdraw their own interest. Maybe just "Withdrawn from HCAP"?